dogpile/cache/region.py

```diff
@@ -879,6 +882,7 @@ def get_value():
                     value.metadata["ct"]
                 )
             ):
+                log.info('Missing cache for key: %s', orig_key)
```

Comment:
I'd prefer log.debug(), there are already many log.debug() lines in lock.py which I believe would already show you that a cache miss occurred, have you tried that?

Reply:
**mike bayer** (zzzeek) wrote:

if we are using orig_key which could be any kind of object, I think we shoudl be calling %r here and not %s.  stringify can often be something expensive and arbitrary whereas %r will show what the object is.  if it's a string, then you see the string repr.

Reply:
Effectively I see them, but they are relay debug message, this one is for me at a higher level...
If you don't agree I will set it at debug.

Reply:
the problem is that a web app at high scale may have *a lot* of these.   info logging usually seems to me like a several-per-minute thing.   debug is a hundreds per minute thing.   I'll grant this is borderline.   this is why if we implement general logging for all the major points here it will perhaps be clearer what it should look like.